cognite/replicator/datapoints.py

```diff
@@ -125,6 +146,22 @@ def replicate_datapoints(
                     transformed_values.append(transformed_datapoint.value)
                 datapoints = Datapoints(timestamp=transformed_timestamps, value=transformed_values)
 
+            if value_manipluation_lambda_fnc:
+                transformed_values = []
+                transformed_timestamps = []
+                lambda_fnc = evaluate_lambda_function(value_manipluation_lambda_fnc)
+                if lambda_fnc:
+                    for src_datapoint in datapoints:
+                        try:
+                            transformed_datapoint = src_datapoint
+                            transformed_datapoint.value = lambda_fnc(src_datapoint.value)

```

Comment:
Is this necessary? Seems like you can append the lambda_fnc(src...) to the transformed_values directly :)

Reply:
Yes, you are right, I just approved merge request without seeing this. Should I take it back?